dev/create-release/spark-rm/Dockerfile

```diff
@@ -15,16 +15,20 @@
 # limitations under the License.
 #
 
-# Image for building Spark releases. Based on Ubuntu 16.04.
+# Image for building Spark releases. Based on Ubuntu 18.04.
 #
 # Includes:
 # * Java 8
 # * Ivy
-# * Python/PyPandoc (2.7.12/3.5.2)
-# * R-base/R-base-dev (3.3.2+)
+# * Python/PyPandoc (2.7.15/3.6.7)
+# * R-base/R-base-dev (3.5.0+)
```

Comment:
I'm not sure - particularly whether that brings in packages that we haven't been testing with. I guess we will find out in the next RC

Reply:
FYI, we don't run R 3.5+ in Jenkins or any where

Reply:
In Ubuntu 18.04LTS, R 3.3 is no longer supported. Only R 3.5 is supported.